docs/gen-ai/llm-spans.md

```diff
@@ -0,0 +1,84 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM Calls
+--->
+
+# Semantic Conventions for LLM requests
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+- [LLM Request attributes](#llm-request-attributes)
+- [Events](#events)
+
+<!-- tocstop -->
+
+A request to an LLM is modeled as a span in a trace.
+
+The **span name** SHOULD be set to a low cardinality value describing an operation made to an LLM.
+For example, the API name such as [Create chat completion](https://platform.openai.com/docs/api-reference/chat/create)
```

Comment:
```suggestion
For example, the API name such as [Create chat completion](https://platform.openai.com/docs/api-reference/chat/create) could be represented as `ChatCompletions gpt-4` to represent the API and the LLM.
```
I think completing this sentence with an example would help. In a future PR, we could include additional examples.